### DISCUSSION TITLE
Up arrow only goes back one history item

### DISCUSSION THREAD

**Discussion by drcongo** [ohmyzsh/ohmyzsh#10829] [Help]
We have an Ansible role that installs ohmyzsh on our servers, and even though the config in this role hasn't changed, for the last couple of months, every server we provision has a bug where hitting the up arrow only goes back one history item and then nothing happens if I hit it again. I never realised how much I relied on that arrow key until it stopped working. Does anyone know if something changed in zsh or ohmyzsh that could cause this?

This is basically the entirety of our `zshrc`...

```
ZSH=$HOME/.oh-my-zsh
ZSH_THEME="hactar"
plugins=(git autojump fabric pip python wd zsh-autosuggestions)

# Alias to make the cursor magically re-appear
alias cursor="printf '\e[?25h'"

source $ZSH/oh-my-zsh.sh

# Customize to your needs...
export PATH=$PATH:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
export SERVER_ENV="staging"

ZSH_AUTOSUGGEST_STRATEGY=history

alias octal="stat -c '%a'"

source /usr/share/autojump/autojump.sh
```

**Comment by mcornella** [comment]
Let's debug this:
1. What do you get after pressing <kbd>CTRL-V</kbd> and then <kbd>Up Arrow</kbd>?
2. What do you get after running `bindkey "<output-of-question-1>"`?

**Comment by drcongo** [comment]
Hi @mcornella 

1. it outputs ` ^[OA`
2. `"^[OA" up-line-or-beginning-search`

**Reply by mcornella** [reply]
Let's continue in https://github.com/ohmyzsh/ohmyzsh/issues/10831 because otherwise it's going to be a mess.
